trim_adapters.fa

```diff
@@ -0,0 +1,54 @@
+>PrefixNX/1
```

Comment:
Ideally, Git should not be used to store bioinformatics data but in this case, it makes sense 👍